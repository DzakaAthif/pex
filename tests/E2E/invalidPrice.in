BUY 0 Avocado 50 0;
BUY 0 Avocado 50 -1;
BUY 0 Avocado 50 -100;
BUY 0 Avocado 50 1000000;
BUY 0 Avocado 50 abc;
BUY 0 Avocado 50 100.5;
SELL 0 Avocado 50 0;
SELL 0 Avocado 50 -1;
SELL 0 Avocado 50 -100;
SELL 0 Avocado 50 1000000;
SELL 0 Avocado 50 abc;
SELL 0 Avocado 50 100.5;
BUY 0 Avocado 50 100;
AMEND 0 50 0;
AMEND 0 50 -1;
AMEND 0 50 -100;
AMEND 0 50 1000000;
AMEND 0 50 abc;
AMEND 0 50 100.5;
