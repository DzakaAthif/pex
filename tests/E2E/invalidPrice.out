[PEX] Starting
[PEX] Trading 2 products: Wine Avocado
[PEX] Created FIFO /tmp/pe_exchange_0
[PEX] Created FIFO /tmp/pe_trader_0
[PEX] Starting trader 0 (./trader1)
[PEX] Connected to /tmp/pe_exchange_0
[PEX] Connected to /tmp/pe_trader_0
[PEX] [T0] Parsing command: <BUY 0 Avocado 50 0>
[PEX] [T0] Parsing command: <BUY 0 Avocado 50 -1>
[PEX] [T0] Parsing command: <BUY 0 Avocado 50 -100>
[PEX] [T0] Parsing command: <BUY 0 Avocado 50 1000000>
[PEX] [T0] Parsing command: <BUY 0 Avocado 50 abc>
[PEX] [T0] Parsing command: <BUY 0 Avocado 50 100.5>
[PEX] [T0] Parsing command: <SELL 0 Avocado 50 0>
[PEX] [T0] Parsing command: <SELL 0 Avocado 50 -1>
[PEX] [T0] Parsing command: <SELL 0 Avocado 50 -100>
[PEX] [T0] Parsing command: <SELL 0 Avocado 50 1000000>
[PEX] [T0] Parsing command: <SELL 0 Avocado 50 abc>
[PEX] [T0] Parsing command: <SELL 0 Avocado 50 100.5>
[PEX] [T0] Parsing command: <BUY 0 Avocado 50 100>
[PEX]	--ORDERBOOK--
[PEX]	Product: Wine; Buy levels: 0; Sell levels: 0
[PEX]	Product: Avocado; Buy levels: 1; Sell levels: 0
[PEX]		BUY 50 @ $100 (1 order)
[PEX]	--POSITIONS--
[PEX]	Trader 0: Wine 0 ($0), Avocado 0 ($0)
[PEX] [T0] Parsing command: <AMEND 0 50 0>
[PEX] [T0] Parsing command: <AMEND 0 50 -1>
[PEX] [T0] Parsing command: <AMEND 0 50 -100>
[PEX] [T0] Parsing command: <AMEND 0 50 1000000>
[PEX] [T0] Parsing command: <AMEND 0 50 abc>
[PEX] [T0] Parsing command: <AMEND 0 50 100.5>
[PEX] Trader 0 disconnected
[PEX] Trading completed
[PEX] Exchange fees collected: $0
